haproxy.cfg.tpl

```diff
@@ -36,6 +36,20 @@ defaults
   # Timeout for health check
   timeout check    5s
 
+% if statusbind:
+listen stats
+  bind ${statusbind}
+  mode http
+  stats enable
```

Comment:
I'd add `stats refresh 5s` (or some other value) to automatically refresh the ui every x seconds. You can always turn it off with a click of a button.